### ISSUE TITLE
(bug) No space detected where there should be a single space

### ISSUE THREAD

**Issue by alec-c4** [Shopify/erb-lint#312]
Hey!
I've met a strange bug. When I run erb-lint from my pre-commit hook with 

```
bundle exec erblint --lint-all --autocorrect
```

I see following errors

```
  EXECUTE > erb-lint
Linting 70 files with 13 linters...

No space detected where there should be a single space.
In file: app/views/layouts/_application_header.html.erb:18

No space detected where there should be a single space.
In file: app/views/layouts/_application_header.html.erb:95

No space detected where there should be a single space.
In file: app/views/layouts/_application_header.html.erb:133

No space detected where there should be a single space.
In file: app/views/layouts/_application_header.html.erb:137

No space detected where there should be a single space.
In file: app/views/layouts/_application_header.html.erb:151

No space detected where there should be a single space.
In file: app/views/layouts/_application_header.html.erb:219

No space detected where there should be a single space.
In file: app/views/layouts/_application_header.html.erb:238

7 error(s) were found in ERB files
```

but when I run 

```
bundle exec erblint --lint-all --autocorrect
```

from console, I see

```
❯ bundle exec erblint --lint-all --autocorrect
Linting and autocorrecting 70 files with 13 linters (10 autocorrectable)...

No errors were found in ERB files
```

There are one of problematic lines

```
<path stroke-linecap="round" stroke-linejoin="round" d="M14.857 17.082a23.848 23.848 0 005.454-1.31A8.967 8.967 0 0118 9.75v-.7V9A6 6 0 006 9v.75a8.967 8.967 0 01-2.312 6.022c1.733.64 3.56 1.085 5.455 1.31m5.714 0a24.255 24.255 0 01-5.714 0m5.714 0a3 3 0 11-5.714 0" />

```

and I have no idea what's wrong with this line
